components/common/images-column/ImagesColumn.tsx

```diff
@@ -0,0 +1,40 @@
+import type { FC } from 'react';
+
+import Container from '@mui/material/Container';
+import Stack from '@mui/material/Stack';
+
+import type { ProjectImageDetails } from '../../../models';
+
+import { ImageContainer } from './styled-components';
+
+interface ImagesColumnProps {
+  data: ProjectImageDetails;
+}
+
+const ImagesColumn: FC<ImagesColumnProps> = ({ data }) => {
+  return (
+    <Container maxWidth="lg" sx={{ marginBottom: { xs: '60px', md: '112px' } }}>
+      <Stack direction={{ xs: 'column', md: 'row' }} spacing={{ xs: '24px', md: '30px' }}>
+        {data.imageSection.map((item, index) => {
+          return (
+            <ImageContainer key={index}>
```

Comment:
Let's use `ImageWrapperComponent` here. It looks like it is something that will help us to remove duplication and avoid the [WET anti-pattern](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself#WET)